src/getHumanAge.test.js

```diff
@@ -7,4 +7,21 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('animals` human age should be 0 if their own age < 15', () => {
+    expect(getHumanAge(0, 14)).toEqual([0, 0]);
+  });
```

Comment:
The test description and expectation seem correct for ages less than 15, assuming the function is supposed to return 0 for both animals. Ensure that the `getHumanAge` function aligns with this logic.

Reply:
done